### PR TITLE
Log and Util modules added

### PR DESCRIPTION
**Pull request by turtledreams**

### CODE REVIEW

**Comment by turtledreams** [comment]
image

**Comment by turtledreams** [comment]
[[RN] Logger and Utils](https://trello.com/c/8meVQGa7/1147-rn-logger-and-utils)

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,4 @@ android/android.iml
 android/.settings/org.eclipse.buildship.core.prefs
 android/.classpath
 countly-sdk-react-native-bridge-*.tgz
+example/AwesomeProject
```
Comment (turtledreams) [inline .gitignore:22]: to not pollute changes when we create an app

File: CountlyConfig.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { initialize } from './Logger.js';
```
Comment (turtledreams) [inline CountlyConfig.js:1]: here I imported the initialize function from new module

File: Logger.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    }
+}
+
+/**
+ * Debug - this should contain logs from the internal workings of the SDK and it's important calls.
+ * This should include things like the SDK configuration options, success or fail of the current network request,
+ * "request queue is full" and the oldest request get's dropped, etc.
+ * @param {String} message
+ */
+function d(message) {
```
Comment (ijunaid) [inline Logger.js:50]: waoooow function 'd' :)

**Pull request merged**